app/controllers/admin/organizations_controller.rb

```diff
@@ -25,6 +25,7 @@ def new
 
   def create
     @organization = Organization.create(organization_params)
+    @organization.users.try(:last).try { |user| user.password = SecureRandom.uuid }
```

Comment:
I think you can be a bit more assertive here -- if `@organization` doesn't already have a user assigned to it, there is a bigger problem at-hand (no one would be able to sign in :) ) and it's OK if it blows up here. I really like the `SecureRandom.uuid` approach for the password, great thinking!  (I realize that we also did the double-try thing on line 31; feel free to update that, if you want!)

```suggestion
    @organization.users.last { |user| user.update(password: SecureRandom.uuid) }
```
Using `update` here ensures that the change is persisted. (It also encrypts the password and saves it to the `password_confirmation` field as well)

Reply:
Done!!